src/Tickets/Seating/app/blockEditor/index.js

```diff
@@ -1,2 +1,3 @@
 import './store';
 import './filters.js';
+import './capacity-table';
```

Comment:
Do not import this file, import only the `filters.js` one, from that import the function that will hook on the filter.